### ISSUE TITLE
pndev update issue

### ISSUE THREAD

**Issue by typeoneerror** [PrecisionNutrition/pndev#17]
```
~/Dev/pn/eternal-sledgehammer (master) $ pndev update
Error: ConfigError: `bin_name` required
```

```
~/Dev/pn/eternal-sledgehammer (master) $ pndev --version
pndev 0.1.20
```

Oddly I seem to have two config files that are different. Which one is the correct one with this version?

```
~/Dev/pn/eternal-sledgehammer (master) $ cat ~/.pndev_config
install_path=$HOME/Dev/pn
~/Dev/pn/eternal-sledgehammer (master) $ cat ~/.pndev_config.toml
install_path = 'DEV/PN'
```

**Comment by ghedamat** [comment]
Ha. Will fix. I removed the bin but probably made a mistake 


The conf file is the one in the directory. With a.toml extension

**Comment by ghedamat** [comment]
fixed in 0.1.20

you will have to manually install latest though
